website/sidebars.js

```diff
@@ -192,12 +192,7 @@ module.exports = {
       label: "Migration guides",
       collapsible: false,
       items: [
-        {
-          type: "link",
-          label: "Migrating StateNotifier/ChangeNotifier to Notifier (WIP)",
-          href: "https://github.com/rrousselGit/riverpod/tree/master/examples/marvel",
-        },
-
+        "migration/1.0.0_to_2.0.0",
```

Comment:
This wasn't support to be a v1->v2 page but a Notifier-specific page

Reply:
I'm fine having a v1->v2 page, but IMO that should be a separate page

Reply:
Perfect. We can address the v1 --> v2 in a separate PR at this point. There's no tracking issue about that, either, so I don't want to add confusion on this PR.